api/src/integrations/core/datasource.js

```diff
@@ -384,6 +385,14 @@ class CoreDataSource {
       const promises = preparedPermits.map(permit => permitUtils.createItem(permit));
       await Promise.all(promises);
     }
+
+    // get permit info from transformed core record to ensure permittee + permit num are updated
+    const permitToUpdate = await this.getMinePermit(nrptiRecord);
```

Comment:
I don't think this line is needed since it should have the same value as the existingPermit on line 351.  So `nrptiRecord` isn't also needed in this function

Reply:
Oh that's right, removed this extra lookup.